integration_tests/e2e/prisonerChangeIncentiveLevelDetails.cy.ts

```diff
@@ -0,0 +1,122 @@
+import Page from '../pages/page'
+import PrisonerChangeIncentiveLevelDetailsPage from '../pages/prisonerIncentiveLevels/prisonerChangeIncentiveLevelDetailsPage'
+import PrisonerChangeIncentiveLevelConfirmationPage from '../pages/prisonerIncentiveLevels/prisonerChangeIncentiveLevelDetailsConfirmationPage'
+
+const IncentiveSummary = {
```

Comment:
would it be easier to reuse [sampleReviewHistory](https://github.com/ministryofjustice/hmpps-incentives-ui/blob/main/server/testData/incentivesApi.ts#L81)? cos the type here isn't specified and it would be easier to change sample data in one place